getting-started/python/index.md

```diff
@@ -1,4 +1,4 @@
-# Python Client
+# Get started with openEO Python Client
```

Comment:
That should be streamlined with the other clients (either with the prefix or without)

Reply:
Ok, I'll add it for the other clients